Tools/jenkins/build.dockerfile

```diff
@@ -7,13 +7,10 @@ ARG platform_image="platforms;${emulator_platform}"
 RUN apt-get update && \
   apt-get install -y --no-install-recommends \
   nodejs=6.11.4~dfsg-1ubuntu1 \
-  npm \
```

Comment:
we needed this because the above nodejs package doesnt for some reason have npm last i tested it. And npm is what we use to install yarn, which is what is needed by our scripts.